sass/_content.scss

```diff
@@ -5,287 +5,319 @@
 
-        a {
-            color: $text-color-heading;
-            padding: 10px 0;
-            position: relative;
+    a {
+      color: $text-color-heading;
+      padding: 10px 0 0;
+      position: relative;
 
-            &:hover {
-                border-color: transparent;
-                color: $link-color;
-            }
-        }
+      &:hover {
+        border-color: $link-border-color;
+        color: $link-color;
+      }
     }
+  }
 
-    &-subtitle {
-        @include rem(font-size, 20px);
-        color: #999;
-        margin: 0;
+  &-subtitle {
+    @include rem(font-size, 20px);
+    color: #999;
+    margin: 0;
 
-        a {
-            color: #999;
+    a {
+      color: #999;
 
-            &:hover {
-                color: $link-color;
-            }
-        }
+      &:hover {
+        border-color: $link-border-color;
+        color: $link-color;
+      }
     }
+  }
 
-    &-summary {
-        > * {
-            @include site-width;
-        }
+  &-summary {
+    margin: 1.5em 0 0;
+    position: relative;
 
-		p {
-            margin: 1.5em auto 0;
+    > * {
+      @include site-width;
+    }
 
-            &:first-child {
-                margin-top: 0;
-            }
-        }
+    p {
+      margin: 1.5em auto 0;
 
-        .tiled-gallery {
-            margin-top: 30px;
-        }
+      &:first-child {
+        margin-top: 0;
+      }
     }
 
-    &-content {
-
-        > h1,
-        > h2,
-        > h3,
-        > h4,
-        > h5,
-        > h6,
-        > div,
-        > span,
-        > p,
-        > ul,
-        > ol,
-        > blockquote,
-        > hr,
-        > form,
-        > iframe,
-        > address,
-        > table,
-        > dl {
-            @include site-width;
-        }
+    .tiled-gallery {
+      margin-top: 30px;
+    }
+  }
+
+  &-content {
+    > h1,
+    > h2,
+    > h3,
+    > h4,
+    > h5,
+    > h6,
+    > div,
+    > span,
+    > p,
+    > ul,
+    > ol,
+    > blockquote,
+    > hr,
+    > form,
+    > iframe,
+    > address,
+    > table,
+    > dl {
+      @include site-width;
+    }
 
-        > ul,
-        > ol {
-            li {
-                margin-left: 25px;
-            }
-
-            &.inline {
-                li {
-                    display: inline-block;
-                    margin-left: 0;
-                    margin-right: 10px;
-
-                    &:after {
-                        color: #ccc;
-                        content: '\2022';
-                        display: inline-block;
-                        padding-left: 10px;
-                    }
-
-                    &:last-child:after {
-                        display: none;
-                    }
-                }
-            }
+    > ul,
+    > ol {
+      li {
+        margin-left: 25px;
+      }
+
+      &.inline {
+        li {
+          display: inline-block;
+          margin-left: 0;
+          margin-right: 10px;
+
+          &:after {
+            color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.